migrations/00161_achieved_items.sql

```diff
@@ -0,0 +1,151 @@
+-- +goose Up
+/***********************************************************/
+/*** SCRIPT AUTHOR: Fredrik Vedvik (fredrik@vedvik.tech) ***/
+/***    CREATED ON: 2023-03-24T10:34:47.688Z             ***/
+/***********************************************************/
+
+--- BEGIN CREATE SEQUENCE "public"."achievementconditions_studytopics_id_seq" ---
+
+
+CREATE SEQUENCE IF NOT EXISTS "public"."achievementconditions_studytopics_id_seq"
+	INCREMENT BY 1
+	MINVALUE 1
+	MAXVALUE 2147483647
+	START WITH 1
+	CACHE 1
+	NO CYCLE;
+
+GRANT SELECT ON SEQUENCE "public"."achievementconditions_studytopics_id_seq" TO directus; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT USAGE ON SEQUENCE "public"."achievementconditions_studytopics_id_seq" TO directus; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT UPDATE ON SEQUENCE "public"."achievementconditions_studytopics_id_seq" TO directus; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+
+COMMENT ON SEQUENCE "public"."achievementconditions_studytopics_id_seq"  IS NULL;
+
+--- END CREATE SEQUENCE "public"."achievementconditions_studytopics_id_seq" ---
+
+--- BEGIN CREATE TABLE "public"."achievementconditions_studytopics" ---
+
+CREATE TABLE IF NOT EXISTS "public"."achievementconditions_studytopics" (
+	"id" int4 NOT NULL DEFAULT nextval('achievementconditions_studytopics_id_seq'::regclass) ,
+	"achievementconditions_id" uuid NULL  ,
+	"studytopics_id" uuid NULL  ,
+	CONSTRAINT "achievementconditions_studytopics_pkey" PRIMARY KEY (id) ,
+	CONSTRAINT "achievementconditions_studytopics_studytopics_id_foreign" FOREIGN KEY (studytopics_id) REFERENCES studytopics(id) ON DELETE SET NULL ,
+	CONSTRAINT "achievementconditions_studytopics_achievem__2e3395b2_foreign" FOREIGN KEY (achievementconditions_id) REFERENCES achievementconditions(id) ON DELETE SET NULL
```

Comment:
I'm just sort of unsure about all of this being nullable. 
Is this actually needed for some DU stuff or can we cascade it + make it not nullable?